Dockerfile.build.template

```diff
@@ -0,0 +1,37 @@
+# Build nodejs dependencies
+FROM resin/%%ARCH%%-node:6.2-slim
```

Comment:
The node version here has to match the node version in the oe-image. I don't want to use slim for this because it's a build image and I'd rather install as less packages as possible